src/main/java/taxi/util/Encrypt.java

```diff
@@ -0,0 +1,26 @@
+package taxi.util;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.KeySpec;
+import java.util.Base64;
+import javax.crypto.SecretKeyFactory;
+import javax.crypto.spec.PBEKeySpec;
+
+public class Encrypt {
+    private static final String SALT = "ENyeheAGgPK+Ifj95S7dNQ==";
```

Comment:
Is static salt a good practice? Maybe better do it autogenerated and save it as a driver field?